index.html

```diff
@@ -11,22 +11,214 @@
+                            <ul class="language">
+                            
+                            <li><button type="button">Ruby on rails</button></li>
+                            <li><button type="button">css</button></li>
+                            <li><button type="button">Java Script</button></li>
+                            <li><button type="button">HTML</button></li>
+                        </ul>
+                        </div>
+                    </div>
+                    <div class="box2">
+                        <div class="skills">
+                            <img src="images/icon-frameworks.svg" alt="icon Frameworks" class="icon">
+                            <h2 class="skills">Frameworks</h2>
+                            
+                            <ul class="language">
+                            <li><button type="button">Bootstrap</button></li>
+                            <li><button type="button">Ruby on rails</button></li>
+                            <li><button type="button">RSpec</button></li>
+                            <li><button type="button">Capybara</button></li>
+                            <li><button type="button">Selenium</button></li>
+                        </ul>
+                        </div>
+                    </div>
+                    <div class="box3">
+                        <div class="skills">
+                            <img src="images/icon-skills.svg" alt="icon skills" class="icon">
+                            <h2 class="skills">Skills</h2>
+                            <ul class="language">
+                            <li><button type="button">Codekit</button></li>
+                            <li><button type="button">GitHub</button></li>
+                            <li><button type="button">Codepen</button></li>
+                            <li><button type="button">Gitlab</button></li>
+                            <li><button type="button">Terminal</button></li>
+                        </ul>
+                        </div>        
+                    </div>
+                </div>
+            <br>
+            <br>
+            <br>
+            <div class="aboutyou">
+                <div class="comment"><h2 class="aboutyou-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2></div>
+                <div class="form"><form action="https://formspree.io/f/mbjwboje" method="post">
+                    <input class="textarea" type="text" placeholder="name" maxlength="30" required>
+                    <input class="textarea" type="email" placeholder="email" required>
+                    <textarea class="textarea" name="mdr_message" id="message" cols="50" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
+                   <button class="projects" type="submit">Get in touch
+                        </button>
+                </form></div>
+            </div>
```

Comment:
- [ ] Kindly stick to the design of the template for the form section based on the previous reviewer's suggestions. The form is overlapping the background image in your design and that is not the case for the design template. Kindly implement it. Also, in the design, the Get in Touch button is aligned to the left and not the centre.

### Your Design

![image](https://user-images.githubusercontent.com/77106962/172943904-c0c08b5b-2ca6-4b1c-a815-4c8af424627a.png)

### Figma Design

![image](https://user-images.githubusercontent.com/77106962/172944007-a575718b-75ef-4679-ad18-b47e7e100d3b.png)

#### Kindly make this section identical to the template design 👍